outputs.tf

```diff
@@ -0,0 +1,68 @@
+output "azurerm_kubernetes_cluster_main_object_id" {
+  description = "value of the object id of the service principal"
+  value = azurerm_kubernetes_cluster.main.identity[0].principal_id
+  sensitive   = true
+}
+
+output "azurerm_kubernetes_cluster_main_kube_config_host" {
+  description = "value of the host of the kube config"
+  value = azurerm_kubernetes_cluster.main.kube_config.0.host
+  sensitive   = true
+}
+
+output "azurerm_kubernetes_cluster_main_kube_config_client_certificate" {
+  description = "value of the client certificate of the kube config"
+  value = azurerm_kubernetes_cluster.main.kube_admin_config.0.client_certificate
+  sensitive   = true
+}
+
+output "azurerm_kubernetes_cluster_main_kube_config_client_key" {
+  description = "value of the client key of the kube config"
+  value = azurerm_kubernetes_cluster.main.kube_admin_config.0.client_key
+  sensitive   = true
+}
+
+output "azurerm_kubernetes_cluster_main_kube_config_cluster_ca_certificate" {
+  description = "value of the cluster ca certificate of the kube config"
+  value = azurerm_kubernetes_cluster.main.kube_admin_config.0.cluster_ca_certificate
+  sensitive   = true
+}
+
+output "azurerm_kubernetes_cluster_main_kube_config_cluster_raw" {
+  description = "value of the cluster raw of the kube config"
+  value     = azurerm_kubernetes_cluster.main.kube_config_raw
+  sensitive = true
+}
+
+output "aks_id" {
+  description = "The `azurerm_kubernetes_cluster`'s id."
+  value       = azurerm_kubernetes_cluster.main.id
+}
+
+output "aks_name" {
+  description = "The `aurerm_kubernetes-cluster`'s name."
+  value       = azurerm_kubernetes_cluster.main.name
+}
+
+output "azure_policy_enabled" {
```

Comment:
Here we're just passing the variable value right? I don't think we need this output.